test/customHarvesterInstance.spec.js

```diff
@@ -0,0 +1,39 @@
+var request = require('supertest');
+var should = require('should');
+var harvester = require('../lib/harvester');
+
+var config = require('./config.js');
+var seeder = require('./seeder.js');
+
+/**
+ * This test case demonstrates how to setup test with custom harvester on different port
+ */
+describe('Custom harvester demo', function () {
+    var baseUrl = 'http://localhost:8001';
+    before(function () {
+        function customAppConfigurator(app) {
+            app.resource('pets', {
+                name: String
+            });
+        }
+
+        var app = harvester(config.harvester.options);
```

Comment:
remember we want to call it harvesterApp rather than app, because it's not express.